pkg/systemstatsmonitor/memory_collector_unix.go

```diff
@@ -58,6 +58,29 @@ func (mc *memoryCollector) collect() {
 		}
 	}
 
+	if mc.mPercentUsed != nil && meminfo.MemTotal != nil && *meminfo.MemTotal > 0{
```

Comment:
Why do we need so many percentage memory metrics (and only for Unix)? Shouldn't it just be "used"?

Reply:
That is a copy from mBytesUsed section.
Personally I don't find much use for other than 'used' metrics and I am happy to remove them.

Reply:
Yes, please remove it. Thanks!

Reply:
@mmiranda96 I made the changes, please check.